groupdocs-total/GroupDocs.Total-for-NET-MVC
# Removing outdated jquery package references.

**Problem:**
1. After moving our apps from JQuery to Angular UI we now don't need references to jquery npm packages.
2. We decided to response with code `InternalServerError` from the catch blocks instead of `OK`.

**Solution:**
1. References to the jquery npm packages were removed except of on for `Total`, as i can understand it is used for showing the home page.
2. All found `OK` response statuses inside the catch blocks were replaced by `InternalServerError`.

**Tests:**
1. Build&Run.
2. Check the homepage (locally it is `localhost:8080`):
![image](https://user-images.githubusercontent.com/17431807/74450584-574a5300-4e8f-11ea-9a73-93dd06f97e54.png)
3. Check that all available apps are opening without errors.
